msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -70,6 +86,75 @@ Resource-specific consent (RSC) is a Microsoft Teams and Graph API integration t
    }
 ```
 
+### Example for RSC Delegated permissions
+
+```json
+"authorization": {
```

Comment:
This seems like a new property in manifest. This needs update in Manifest docs as well.